pwnlib/heap/glmalloc/arena/arena.py

```diff
@@ -0,0 +1,116 @@
+from pwnlib.heap.glmalloc.bins import NoTcacheError
+from pwnlib.heap.glmalloc.malloc_state import \
+    SMALL_BINS_START_INDEX, \
+    LARGE_BINS_START_INDEX
+from pwnlib.heap.glmalloc.basic_formatter import BasicFormatter
+
+
+class Arena(object):
+    """Class with the information of the arena.
+    """
+
+    def __init__(self, malloc_state, heap, unsorted_bin,
```

Comment:
Needs some kind of test. I recommend using `pwnlib.data` and adding a `Corefile` that you can extract data from.

Reply:
Tests were added in HeapExplorer that uses this class in 72d99e4